packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -109,13 +118,13 @@ const ResultsPagination: FC<ResultsPaginationProps> = ({
         {!isResultsPaginationComponentHidden && (
           <CarbonPagination
             className={classNames.join(' ')}
-            page={page}
+            page={currentPage}
             totalItems={matchingResults}
             pageSize={actualPageSize}
             pageSizes={pageSizes}
             onChange={handleOnChange}
             itemRangeText={handleItemRangeText}
-            itemsPerPageText={mergedMessages.itemsPerPageText}
+            itemsPerPageText={'YAY'}
```

Comment:
I think this should still be `mergedMessages.itemsPerPageText` and is also why some tests are failing...:)

Reply:
doh :(